### PR TITLE
Preserve CSP protection when upstream scripts emit empty nonces

### PR DESCRIPTION
**Pull request by kwakayama**
## Summary
- replace empty or whitespace-only `nonce` attributes with the response CSP nonce during HTML post-processing
- keep non-empty nonce attributes untouched
- add a regression test covering `nonce=""` and whitespace-only nonce values

## Root Cause
Production preview HTML can include an inline `next-themes` bootstrap script rendered as `<script nonce="">...</script>`. The shared Veryfront nonce injector treated any existing `nonce=` attribute as authoritative, so the empty value survived HTML post-processing even though the response CSP required a concrete nonce.

That leaves the page in an inconsistent state:
- CSP header: `script-src 'self' 'nonce-<response-nonce>' ...`
- HTML body: inline script with `nonce=""`

Browsers block that inline script, which is the CSP error seen on the `Agent Creator Widget` preview repro.

Related issue: veryfront/veryfront-studio#1207

## Verification
- `deno test --no-check --allow-all src/html/nonce-injection.test.ts`
- `deno check src/html/nonce-injection.ts src/html/nonce-injection.test.ts`
- `deno lint src/html/nonce-injection.ts src/html/nonce-injection.test.ts`

## Risks
- Not yet verified against a deployed production preview after rollout

### CODE REVIEW

File: src/html/nonce-injection.ts [hunk 1]
```diff
@@ -56,7 +56,16 @@ function findRawTextClosingTagStart(
 }
 
 function injectNonceIntoOpeningTag(tag: string, escapedNonce: string): string {
-  if (/(?:\s|<)nonce\s*=/iu.test(tag)) return tag;
+  const existingNonce = /(\snonce\s*=\s*)(?:"([^"]*)"|'([^']*)'|([^\s>]+))/iu.exec(tag);
```
[the remark this answers is not in the document]
Comment (kwakayama) [reply src/html/nonce-injection.ts]: Addressed in 149de1585. I replaced the raw regex with a quote-aware attribute scanner so `nonce=` text inside another attribute value is ignored, and added a regression test for that case.

File: src/html/nonce-injection.ts [hunk 2]
```diff
@@ -56,7 +56,16 @@ function findRawTextClosingTagStart(
...
+  if (existingNonce) {
+    const currentValue = existingNonce[2] ?? existingNonce[3] ?? existingNonce[4] ?? "";
+    if (currentValue.trim()) return tag;
+
+    const replacement = ` nonce="${escapedNonce}"`;
+    return `${tag.slice(0, existingNonce.index)}${replacement}${
+      tag.slice(existingNonce.index + existingNonce[0].length)
+    }`;
```
[the remark this answers is not in the document]
Comment (kwakayama) [reply src/html/nonce-injection.ts]: Addressed in 149de1585. The injector now parses attributes outside quotes before deciding whether a real `nonce` attribute exists, which avoids corrupting `data-*` values that merely contain `nonce=` text.
